test/verifiedTests/analysis/testMultiSpeciesModelling/testMappingInfo.m

```diff
@@ -0,0 +1,109 @@
+% The COBRAToolbox: testMappingInfo.m
+%
+% Purpose:
+%     - test the mappingInfo function of the Microbiome Toolbox
+%
+% Authors:
+%     - original version: Federico Baldini & Laurent Heirendt, April 2018
+%
+
+global CBTDIR
+
+% define the features required to run the test
+requiredToolboxes = { 'distrib_computing_toolbox' };
+
+requiredSolvers = { 'ibm_cplex' };
+
+% require the specified toolboxes and solvers, along with a UNIX OS
+solversPkgs = prepareTest('requiredSolvers', requiredSolvers, 'requiredToolboxes', requiredToolboxes);
+
```

Comment:
not sure, but I think it would make sense to actually change to ibm_cplex before running things.

Reply:
This is in progress...